app/models.py

```diff
@@ -1462,10 +1464,12 @@ def validate(self, enforce_required=True, required_fields=None, max_day_rate=Non
 
     def serialize(self):
         data = self.data.copy()
+        parent_brief = self.brief.serialize()
+        parent_brief_fields = ['id', 'title', 'status', 'applicationsClosedAt']
```

Comment:
> Why do we restrict ourselves to these fields? 
If we want to refactor in the future it'll be easier to remove just a few fields than the whole serialisation :man_shrugging: 


> I guess the question is whether we need clarification questions in the list of briefs response at all
Probably not but that's out of scope of this ticket.